service/routes/docs.js

```diff
@@ -14,8 +14,19 @@ const compatdata = require('../../docs/assets/compat.json');
 const appVersion = require(path.join(__dirname,'../../package.json')).version;
 
 const cache = {};
-const cachettls = {fastly:1800, respTimes:1800, outages:86400};
+const cachettls = {fastly:1800, respTimes:1800, outages:86400, rumPerf:86400};
 const templCache = {};
+let mysql;
+
+if (process.env.RUM_MYSQL_DSN) {
+	require('promise-mysql')
+		.createConnection(process.env.RUM_MYSQL_DSN)
+		.then(conn => {
+			mysql = conn;
```

Comment:
Which should be fine - it's tested for whenver it is used (because it is also null *until* a connection is made)